configure.ac

```diff
@@ -96,6 +96,17 @@ AC_PROG_CC_C99
 AM_PROG_CC_C_O
 AC_PROG_CPP
 
+# Put this at the bottom of Libfabric's (internal) config.h
+AH_BOTTOM([
+/* Libfabric requires a C99 compiler, and C99 defines the printf
+ * escape for (size_t) to be "zu".  If the compiler doesn't define
+ * PRIsize_t, then define it for ourselves.
+ */
+#if !defined(PRIsize_t)
+#define PRIsize_t "zu"
+#endif
```

Comment:
Can we not just use zu directly?

Reply:
Sure.  I was just making it be analogous to the other `PRIfoo` macros that exist.  Some environments have `PRIsize_t`, some don't; that's why this macro is here.

Do you have a strong opinion?

Reply:
Strong opinion?  No.  But since zu exists, I would prefer to use it directly.  It's unfortunate that there's not a u64 standard

Reply:
Ok, `PRIsize_t` replaced with `zu` everywhere.